src/lib/viewers/office/OfficeViewer.js

```diff
@@ -161,8 +161,12 @@ class OfficeViewer extends BaseViewer {
     setupPDFUrl() {
         const { file } = this.options;
         const pdfRep = getRepresentation(file, 'pdf');
-        const { url_template: template } = pdfRep.content;
-        this.pdfUrl = this.createContentUrlWithAuthParams(template);
+
+        // This occurs in the case of .xlsb files where no pdf rep exists
+        if (pdfRep && pdfRep.content) {
```

Comment:
We have a method that's basically a clone of lodash/get called `getProp` that could help avoid these checks.

Reply:
Updated